packages/api/src/sendPayment/sendPayment.ts

```diff
@@ -16,7 +16,7 @@ export const sendPayment = async (payment: PaymentRequestPayload) => {
   try {
     const message: RequestPaymentMessage = {
       app: GEM_WALLET,
-      type: 'SEND_PAYMENT',
+      type: 'SEND_PAYMENT/V3',
```

Comment:
Could we please rename this to:
```suggestion
      type: 'REQUEST_SEND_PAYMENT/V3',
```